[2021-08-09 20:28:06,045] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_4 2021-08-09T20:27:54.689148+00:00 [queued]>
[2021-08-09 20:28:06,062] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_4 2021-08-09T20:27:54.689148+00:00 [queued]>
[2021-08-09 20:28:06,062] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-08-09 20:28:06,063] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-08-09 20:28:06,063] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-08-09 20:28:06,070] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_4> on 2021-08-09T20:27:54.689148+00:00
[2021-08-09 20:28:06,072] {standard_task_runner.py:52} INFO - Started process 172608 to run task
[2021-08-09 20:28:06,076] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_4', '2021-08-09T20:27:54.689148+00:00', '--job-id', '11', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmp07oudbd8', '--error-file', '/tmp/tmpnf23hja8']
[2021-08-09 20:28:06,077] {standard_task_runner.py:77} INFO - Job 11: Subtask task_4
[2021-08-09 20:28:06,203] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_4 2021-08-09T20:27:54.689148+00:00 [running]> on host airflowvm
[2021-08-09 20:28:06,325] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_4
AIRFLOW_CTX_EXECUTION_DATE=2021-08-09T20:27:54.689148+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-08-09T20:27:54.689148+00:00
[2021-08-09 20:28:06,326] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-08-09 20:28:06,326] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2021-08-09 20:28:06,334] {subprocess.py:75} INFO - Output:
[2021-08-09 20:28:09,337] {subprocess.py:83} INFO - Command exited with return code 0
[2021-08-09 20:28:09,381] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_4, execution_date=20210809T202754, start_date=20210809T202806, end_date=20210809T202809
[2021-08-09 20:28:09,422] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-08-09 20:28:09,449] {local_task_job.py:151} INFO - Task exited with return code 0
